### PR TITLE
Update usaco-491.mdx

### PR DESCRIPTION
**Pull request by Sanjana-Reddy-Kakulavaram**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

File: solutions/usaco-491.mdx [hunk 1]
```diff
@@ -2,14 +2,67 @@
...
 <LanguageSection>
+
+<CPPSection>
+```cpp
+	
+#include <bits/stdc++.h>
+using namespace std;
+unordered_map<int, vector<int>> mp;
+vector<vector<int>> dist;
+void bfs(int s, int ix)
```
Comment (SansPapyrus683) [inline solutions/usaco-491.mdx]: what does this function do?

Comment (Sanjana-Reddy-Kakulavaram) [reply solutions/usaco-491.mdx]: Finds the distance between node s and node ix, I have included a comment defining this in code

Comment (Sanjana-Reddy-Kakulavaram) [reply solutions/usaco-491.mdx]: can you review the code please.

File: solutions/usaco-491.mdx [hunk 2]
```diff
@@ -51,9 +102,9 @@ public class PiggyBack {
 		}
 
 
-		bfs(0, 0);
-		bfs(1, 1);
-		bfs(N-1, 2);
+		distance(0, 0);
+		distance(1, 1);
+		distance(N-1, 2);
```
Comment (SansPapyrus683) [inline solutions/usaco-491.mdx:107]:
```suggestion
		distance(N - 1, 2);
```

File: solutions/usaco-491.mdx [hunk 3]
```diff
@@ -2,14 +2,65 @@
...
+
+<CPPSection>
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+unordered_map<int, vector<int>> mp;
+vector<vector<int>> dist;
+
+// Functipon to find distance from node s to node ix
```
Comment (SansPapyrus683) [inline solutions/usaco-491.mdx:22]: typo in function

**Comment by SansPapyrus683** [comment]
@envyaims review

**Review by envyaims — APPROVED** [review]
ok ig

**Pull request merged**